### DISCUSSION TITLE
How to activate last pane?

### DISCUSSION THREAD

**Discussion by himanshutank** [wezterm/wezterm#4514] [General]
Need help to understand is there any way to activate last pane?

@wez love this terminal 🙏

**Comment by nekowinston** [comment]
[`wezterm.action.ActivateTab(-1)`](https://wezfurlong.org/wezterm/config/lua/keyassignment/ActivateTab.html)
> ActivateTab now accepts negative numbers; these wrap around from the start of the tabs to the end, so -1 references the right-most tab, -2 the tab to its left and so on.

**Reply by himanshutank** [reply]
Hi @nekowinston ,

Yes, that helps for the tabs.

I am looking for panes.

BR,
Himanshu

**Comment by luv2code** [comment]
I opened #3505 for this and I put a workaround in a comment. The workaround doesn't work exactly right; but it meets my needs 70% of the time.

I wish that `ActivateLastPane` was a proper KeyAssignment; but I'd be happy with a `pane_activated` event.
